costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -300,6 +300,7 @@ inline void InflationLayer::enqueue(unsigned int index, unsigned int mx, unsigne
 
 void InflationLayer::computeCaches()
 {
+  ROS_DEBUG_STREAM("Computing caches with " << cell_inflation_radius_ << " radius");
```

Comment:
I thought it would be smart enough not to expand the string unless the log level is high enough, but I'm not sure that is true.  In this case, computeCaches() is not called often (only if the inflation radius changes), so the performance hit isn't bad.  But I will look around to see if there are any other spots.